ospo-book/chapters/01-chapter.md

```diff
@@ -18,3 +24,117 @@ The book is structured in a user-friendly and practical manner, with a focus on
 Whether you're just starting out on your open source journey, or are looking to streamline your existing operations, this book will provide you with the knowledge and tools you need to create and implement a successful Open Source Program Office.
 
 In the following chapters, we will explore the key components of an OSPO, and provide practical guidance and best practices on how to create and implement an OSPO within your organization, regardless of your industry or sector. So let's get started!
+
+## OSPO Definition
+
+An *Open Source Program Office (OSPO)* is a center of expertise, either virtual or physical, that supports, nurtures, shares, explains, and grows open source within an organization. An OSPO *role* can be conceived as an umbrella term, covering the responsibilities for defining and implementing strategies and policies that guide open source efforts and provide tools, processes, and knowledge to organization teams and experts in navigating open source without harming the open source ecosystem. Instead, they help to sustain it while achieving organizational goals.
+
+ OSPOs (as center of expertise) may comprise a framework built around some or all of the following aspects:
+
+* Set policies related to code use, distribution, selection, auditing, and other key areas
+* Provide education and training to internal and external stakeholders involved in open source activities
+* Ensure legal compliance
+* Promote community engagement
+
+OSPO (as a role) can include:
```

Comment:
One role I think is misses is the role that focus on ensuring Open Source Compliancy. My understanding is that their are OSPO that purely focus on that.